app/controllers/events_controller.rb

```diff
@@ -1,4 +1,54 @@
 class EventsController < ApplicationController
+  before_action :authenticate_user!, except: [:index]
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.create_event(event_params)
```

Comment:
- [ ] Uses the **#build** [association refrence method](https://guides.rubyonrails.org/association_basics.html#detailed-association-reference), i.e. `current_user.events.build(event_params)` to create the new event with the user’s ID prepopulated.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))